hoot-core-test/src/test/cpp/hoot/core/elements/RelationTest.cpp

```diff
@@ -102,7 +102,7 @@ class RelationTest : public HootTestFixture
     r2->addElement("", r1->getElementId());
 
     ElementCountVisitor v;
-    r1->visitRw(*map, v);
+    r1->visitRo(*map, v);
 
```

Comment:
This operation was actually read-only, so changed the method call.  There are a handful of these type of changes throughout.